packages/web3-providers-http/types/index.d.ts

```diff
@@ -20,8 +20,6 @@
  * @date 2018
  */
 
-import {HttpProviderBase} from 'web3-core-helpers';
+import { HttpProviderBase } from 'web3-core-helpers';
 
-export {HttpProviderOptions, HttpHeader, JsonRpcPayload, JsonRpcResponse} from 'web3-core-helpers';
-
-export class HttpProvider extends HttpProviderBase { }
+export class HttpProvider extends HttpProviderBase {}
```

Comment:
Those additional exports are required to not import http related types from two places it is just a small improvement without any logical change.

Reply:
I've resolved this and created an issue https://github.com/ethereum/web3.js/issues/3334 for doing it later.